specification/shutdown/delete_node/ShutdownDeleteNodeResponse.ts

```diff
@@ -18,5 +18,7 @@
  */
 
 export class Response {
```

Comment:
You should use `AcknowledgedResponseBase` :)
```suggestion
export class Response extends AcknowledgedResponseBase {}
```